docker/linksets/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.4'
+
+services:
+  linksets:
+    build: 
+      context: linksets-builder 
+      target: python-specific
+    env_file:
+      - ./.env
```

Comment:
Possibly add the common.env file in here too. I was burnt not specifying some of the ENV vars from common in here explicitly, and add the whole env_vile prevents that `- ../../common/common.env`

Reply:
done